package.json

```diff
@@ -14,8 +14,13 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "ch-node-session-handler": "git+ssh://git@github.com/companieshouse/node-session-handler.git#master",
+    "ch-sdk-node": "git+ssh://git@github.com/companieshouse/ch-sdk-node.git",
```

Comment:
I would get the library from a specific tag like this git+ssh://git@github.com/companieshouse/ch-sdk-node.git#7c178fbf5e9ea7ba3c735aa7d93a0dd554d35771 . Going directly from master, means that any changes that are master are in the application, which can cause problems.

Reply:
Good shout, will use the same versions as in certificates